### ISSUE TITLE
OpenAI gpt-4-turbo-preview Model

### ISSUE THREAD

**Issue by adit-s** [deiucanta/chatpad#108]
Hello. First, I'd like to say this is an excellent tool and thanks for writing it. 
I just started using it and it's extremely useful.
I'm new to OpenAI and still learning the concepts. I went through their web pages and know the gpt-4-turbo-preview model is now available via the API and costs less than gpt-4.
I was trying to choose it in Chatpad, but it isn't one of the options (at least that I recognize).

_Is there a way to choose gpt-4-turbo-preview? If not, would you be able to add it as one of the options?_

This [open issue](https://github.com/deiucanta/chatpad/issues/96) would be the perfect solution.

Thanks,
Adit Sood

**Comment by Supernova3339** [comment]
We are currently reviewing current PRs and will be preparing a release
candidate soon.

**Comment by larsilus** [comment]
When will you have a new release? https://platform.openai.com/docs/models/gpt-4-turbo-and-gpt-4
 In April the latest gpt 4 Release is gpt-4-turbo.
Any help needed?

**Comment by bgriffen** [comment]
Seconded, also very interested in being able to use GPT4-turbo, especially owing to the lower costs.

**Comment by jarodsmk** [comment]
Since gpt-4o is available to use via the OpenAI API, it would be great to have that as an included model in an upcoming release too.

**Comment by jarodsmk** [comment]
Created a PR for the above.

**Comment by Supernova3339** [comment]
Will look into it

**Comment by bgriffen** [comment]
Is it really a case of four lines added to a json to activate this? I probably should have looked at the source code earlier. 

@_@

**Comment by jarodsmk** [comment]
From what I could see it seems like it - unless I missed something 👀 (plausible)

**Comment by jarodsmk** [comment]
I gave it another test this morning and noticed a strange quirk, will update my PR. The 3.5/4 model selector at the bottom of the page selects 3.5 despite 4o being selected in the settings menu. I've added 4o as an option, but I think we may need to reconsider that selector in the future.

Here's a sample of 4o working on my current main branch:
![image](https://github.com/deiucanta/chatpad/assets/6736209/ebc3d7e8-edae-4161-a285-0c6976bf799e)

Here's the new option I propose adding, and the model selector I was referring to:
![image](https://github.com/deiucanta/chatpad/assets/6736209/6c488341-8b87-4e92-b85a-36191b0ac85a)
